packages/plugin-core/src/commands/pods/ExportPodV2Command.ts

```diff
@@ -32,6 +33,12 @@ export class ExportPodV2Command extends BaseCommand<
    * undefined if the user didn't select anything.
    */
   async gatherInputs(podId: string): Promise<CommandInput | undefined> {
+    // added check to return if export pod v2 is not enabled in dev config and is run using pod keyboard shortcuts
```

Comment:
fyi: keyboard shortcuts can also take when clauses.

Reply:
see https://github.com/dendronhq/dendron/pull/2190#discussion_r786635567 instead

Reply:
I did try the "enablement" property to the commands contribution point, it does block the command to appear up in the command palette but not the keybinding shortcuts to pod v2.

Reply:
That's weird. I guess it's not blocking any keybindings if the user adds it themselves (only blocking the ones we explicitly put in the contribution points)? We could probably check for the flag before registering the command but I don't think we have to go that far.

Let's just leave it as is and add a comment here to remind ourselves that this check should be removed when we announce it publicly 😄